<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
<suite name="Failed suite [Surefire suite]" verbose="0">
  <test thread-count="5" name="Surefire test(failed)" verbose="0">
    <classes>
      <class name="Selenium.Testpackage.ErrorValidationTest">
        <methods>
          <include name="validatecartProduct"/>
          <include name="validateSignout" invocation-numbers="0 1"/>
          <include name="close"/>
        </methods>
      </class> <!-- Selenium.Testpackage.ErrorValidationTest -->
      <class name="Selenium.Testpackage.StandaloneTest">
        <methods>
          <include name="Full"/>
        </methods>
      </class> <!-- Selenium.Testpackage.StandaloneTest -->
      <class name="Selenium.Testpackage.SubmitOrderTest">
        <methods>
          <include name="submitOrder"/>
          <include name="OrderHistory"/>
          <include name="addOrders"/>
          <include name="close"/>
        </methods>
      </class> <!-- Selenium.Testpackage.SubmitOrderTest -->
    </classes>
  </test> <!-- Surefire test(failed) -->
</suite> <!-- Failed suite [Surefire suite] -->
